ui/src/components/Dashboard/index.tsx

```diff
@@ -1,18 +1,22 @@
 // Dashboard.tsx
-import JobDurationChart from "@app/components/ChartJobDuration";
+'use client';
+import JobDurationChart from '../ChartJobDuration';
 import * as React from 'react';
-import { Card } from '@patternfly/react-core/dist/dynamic/components/Card'
-import { CardBody } from '@patternfly/react-core/dist/dynamic/components/Card'
-import { CardTitle } from '@patternfly/react-core/dist/dynamic/components/Card'
-import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page'
-import { Title } from '@patternfly/react-core/dist/dynamic/components/Title'
-import JobStatusPieChart from '@app/components/ChartJobDistribution';
+import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import { Card } from '@patternfly/react-core/dist/dynamic/components/Card';
+import { CardBody } from '@patternfly/react-core/dist/dynamic/components/Card';
+import { CardTitle } from '@patternfly/react-core/dist/dynamic/components/Card';
+import JobStatusPieChart from '../ChartJobDistribution';
 
-const Dashboard: React.FunctionComponent = () => {
+const Index: React.FunctionComponent = () => {
   return (
     <PageSection>
-      <Title headingLevel="h1" size="lg">Dashboard</Title>
-      <Card className="pf-m-mb-lg"> {/* TODO: the bottom spacing isn't working */}
+      <Title headingLevel="h1" size="lg">
+        Dashboard
+      </Title>
+      <Card>
+        {' '}
```

Comment:
Curious why we need this: `{' '}`

Reply:
residual from trying to add card spacing, removed, ty.